tests/test_network.py

```diff
@@ -1600,3 +1600,16 @@ def test_connect_voltage_level_on_line():
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "voltage_level1_id"] == "N_VL"
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "voltage_level2_id"] == "VLHV2"
     assert retrieved_splittedline2.loc['NHV1_NHV2_1_2', "r"] == 0.75
+
+
+def test_add_load_bay():
+    n = pp.network.create_four_substations_node_breaker_network()
+    df = pd.DataFrame(index=["new_load"], columns=["id", "p0", "q0", "busbar_section_id", "position_order"], data=[["new_load", 10.0, 3.0, "S1VL1_BBS", 0]])
+    pp.network.create_load_bay(network=n, df=df, raise_exception=True)
```

Comment:
Do you think I need to add one with the keywords for each element?

Reply:
Well it would be nice, but maybe we could kind of "automate" it to avoid having to do it by hand, by using the same input dictionary to build a dataframe or keywords, in all tests.

I propose to postpone this, it could be a more general improvement of tests (also for existing methods).